src/python-bindings/classad2_impl.cpp

```diff
@@ -1 +1,48 @@
+#include "condor_python.h"
 
+// For module initialization.
+#include "condor_config.h"
+#include "common2/py_handle.cpp"
+
+// classad.ClassAd
+#include "classad/classadCache.h"
+#include "common2/py_util.cpp"
+
+
+static PyMethodDef classad2_impl_methods[] = {
+	{NULL, NULL, 0, NULL}
+};
+
+
+static struct PyModuleDef classad2_impl_module = {
+	.m_base = PyModuleDef_HEAD_INIT,
+	.m_name = "classad22_impl",
```

Comment:
fix this typo

Reply:
This is fixed at https://github.com/htcondor/htcondor/pull/1958/files#diff-c98f58bb3231d29ce2537a7df7277f119083fb70e22e2356d1fe9c1c0af15e47L19